docusaurus/docs/Android/02-client/06-guides/01-push-notifications/01-overview.mdx

```diff
@@ -35,14 +35,31 @@ ChatClient.Builder("api-key", context)
     .build()
 ```
 
+:::note
+Push notifications are enabled by default on the client-side.
```

Comment:
It is not 100% accurate.
Our class `NotificationsConfig` by default has the property `pushNotificationEnabled` set to `true`, but, because an empty list of `PushDeviceGenerator` is configured (We can't have anyone by default as far as we have provider-implementations on different artifacts), push notifications are not going to be configured by default (SDK won't send device token to the server, and server won't send PN to our SDK)